backend/routes/codes.js

```diff
@@ -3,31 +3,31 @@ const router = express.Router();
 
 //controller functions
 const {
-  getallCodes,
-  getaCode,
-  createaCode,
-  updateacode,
-  deleteacode,
+  getAllCodes,
+  getACode,
+  createACode,
+  updateACode,
+  deleteACode,
 } = require("../controllers/codesController");
 
 const requireAuth = require('../middleware/requireAuth')
 
 router.use(requireAuth)
 
 //get all codes route
-router.get("/getallcodes", getallCodes);
+router.get("/getallcodes", getAllCodes);
 
 //TODO: needs to be updated to work with :id and not be a POST
```

Comment:
Todos need to be removed

Reply:
their might be a few if you use the find tab in vs you should be able to remove all of them